setup.py

```diff
@@ -33,6 +33,8 @@ def get_all_scripts(dirname):
         'tensorboardX>=1.4',
         'pillow>=5.3.0',
         'nara_wpe',
+        'nltk',
+        'kaldiio'
```

Comment:
`'kaldiio'` -> `'kaldiio',` to fix https://travis-ci.org/espnet/espnet/jobs/541274563#L660-L662